localstack/services/edge.py

```diff
@@ -489,6 +489,11 @@ def start_edge(listen_str: str, use_ssl: bool = True, asynchronous: bool = False
     # separate privileged and unprivileged addresses
     unprivileged, privileged = split_list_by(listen, lambda addr: addr.is_unprivileged() or False)
 
+    # if we are root, we can directly bind to privileged ports as well
+    if is_root():
+        unprivileged = unprivileged + privileged
+        privileged = []
```

Comment:
Last time I tried making this change, it broke the -ext pipeline. Can you make sure you do a manual run against this branch.

Reply:
https://github.com/localstack/localstack-ext/pull/2131 running here. Let's hope the double start issue was the culprit last time :)

Reply:
Resolved the issue as discussed, the cause was a double-adding of the port 443 in ext tests.